api/google_analytics_provider.py

```diff
@@ -13,11 +13,22 @@
 )
 
 class GoogleAnalyticsProvider(object):
-    NAME = _("Google Analytics")
 
-    DESCRIPTION = _("<p>Send circulation-related analytics events to Google Analytics." +
-                    "<p>To use this integration, set up a property in Google Analytics. " +
-                    "On the administration page for the property, go to Custom Definitions > Custom Dimensions, " +
+    NAME = _("Google Analytics")
+    DESCRIPTION = _("How to Configure a Google Analytics Integration")
+    INSTRUCTIONS = _("<p>In order to track usage statistics, you can configure the Circulation Manager " +
+                    "to connect to Google Analytics.  Currently, Google Analytics is the only analytics engine " +
+                    "supported for this function.</p>" +
```

Comment:
I don't think this paragraph is necessary. Particularly the last sentence doesn't really need to be on this particular page.

Reply:
I agree about the second sentence; taking it out.  But I think it's nice to have the first sentence as an introduction (particularly for people who haven't used an analytics integration before)...